roadmap/implementors-guide/src/runtime/initializer.md

```diff
@@ -1,16 +1,23 @@
 # Initializer Module
 
-This module is responsible for initializing the other modules in a deterministic order. It also has one other purpose as described above: accepting and forwarding session change notifications.
+This module is responsible for initializing the other modules in a deterministic order. It also has one other purpose as described in the overview of the runtime: accepting and forwarding session change notifications.
 
 ## Storage
 
 ```rust
-HasInitialized: bool
+HasInitialized: bool;
+// buffered session changes along with the block number at which they should be applied.
+//
+// typically this will be empty or one element long. ordered ascending by BlockNumber and insertion
+// order.
```

Comment:
what could cause this to be longer than 1 element?

Reply:
Multiple session-change notifications within a single block or a session change notification after initialization in block X and before initialization in child(X). The latter is more likely in practice, if you have a runtime upgrade that changes module order, but both cases would require a session that is only one block long (very very unlikely in practice)

Reply:
Oh and Governance as mentioned elsewhere